developer-docs-site/static/move_demo/sources/CAPMessageBoard.move

```diff
@@ -0,0 +1,141 @@
+module Examples::CapMessageBoard {
+    use Std::Vector;
+    use Std::Signer;
+    use CoreFramework::Timestamp;
+    use Std::Event::{Self, EventHandle};
+
+    // Error map
+    const ACCOUNT_NO_NOTICE_CAP: u64 = 0;
+    const CANNOT_REMOVE_ADMIN_NOTICE_CAP: u64 = 1;
+
+    struct CapMessageBoard has key, store {
+        notice: vector<u8>
+    }
+    /// delegate the notice update capability,
+    /// the reason having this struct is we cannot directly transfer capability across accounts
+    /// very unintuitive
+    struct NoticeChangeCapabilityContainer has key, store {
```

Comment:
maybe try to leverage Offer.move

Reply:
use offer and the code looks much simplier